microcontroller/Makefile

```diff
@@ -63,7 +63,8 @@ C_INCLUDES = \
 # C defines
 C_DEFS =  \
 	-DUSE_HAL_DRIVER \
-	-DSTM32F767xx
+	-DSTM32F767xx \
+	-DUSE_FULL_ASSERT
```

Comment:
What does this mean?

Reply:
This is a macro defined in STM32 that allows the checking that we passed the arguments to the driver functions. The main reason it is added is that to actually use every parameter (no more unused warnings)